src/Kinepolis/AuthenticatedKinepolisClient.php

```diff
@@ -72,6 +72,34 @@ public function getMovieDetail(string $token, int $mid): array
         return $contents['movies'][0];
     }
 
+    public function getTheaters(string $token): array
```

Comment:
I prefer to call methods that actually fetch information from API's something like "fetchTheaters" instead of the get-prefix.
The get-prefix often communicates that all the information is already in memory, that is not the case here.

Reply:
Would it be better instead of returning an array directly from the API to create a "Theater" DTO that wraps all the info? That way when the API ever changes it's format we don't have to change all the underlying code.
Also will make for better typehints.

Reply:
I think the task of the Client should just be to fetch the Json. The Parsers should Convert the Json to Typed Objects.